geode-lucene/src/test/java/org/apache/geode/cache/lucene/test/LuceneTestUtilities.java

```diff
@@ -79,6 +79,10 @@
       "Cannot create Lucene index index2 on region /region because it is not defined in another member.";
   public static final String CANNOT_CREATE_LUCENE_INDEX_DIFFERENT_INDEXES_3 =
       "Cannot create Lucene index index on region /region because it is not defined in another member.";
+  public static final String CANNOT_CREATE_LUCENE_INDEX_DIFFERENT_SERIALIZER =
+      "Cannot create Lucene index index on region /region with Serializer DummyLuceneSerializer because another member defines the same index with different serializer HeterogeneousLuceneSerializer.";
+  public static final String SERIALIZER_MATCHED =
+      "Serializer matched when creating Lucene index index on region /region.";
```

Comment:
typo, index index

Reply:
This is not typo. The second "index" is the indexname.